### ISSUE TITLE
Structural example 2 (nonlinear plate with continuation solver)

### ISSUE THREAD

**Issue by jdeaton** [MASTmultiphysics/mast-multiphysics#73]
As mentioned in #57, it seems that the load is not being captured during the residual evaluation in structural example 2. This may be due to some of our recent changes, or perhaps there is a command line option that I am not aware of.

**Comment by manavbhatia** [comment]
The default options can be changed so that it runs out of the box. I will look into this.

**Comment by manavbhatia** [comment]
Yes, options in the source code, but ones that can be modified from the command line. I have been using a wrapper to GetPot in all examples. It takes a default value, and can also be modified from command line or an input file. 

The problem with this example is that the default options for the parameters are causing an issue. I did not catch this at my end since I have been running most of these examples with command line options.

**Comment by manavbhatia** [comment]
The libMesh::SolverConfiguration class is specifically for linear/eigen/... solver parameters. This is not the source of issue for this example, I think.
